src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -449,12 +474,14 @@ where
         if msg.kind == b'S' {
             let name = cstr_to_str(&msg.name).unwrap().to_string();
             let statement = if name.is_empty() {
-                &self.unnamed_statement
+                self.unnamed_statement
+                    .as_ref()
+                    .ok_or_else(|| PsqlError::DescribeError("No statement found".to_string()))?
```

Comment:
We can add a method `PsqlError::not_found()` always 
 return PsqlError::DescribeError("No statement found"), so that no need to write the literal message.

ditto for other errors

Reply:
But we also have PsqlError::Bind("No statement found") & PsqlError::Execute("No statement found").
So is weird to provide not_found() only return PsqlError::DescribeError("No statement")?
we can provide`not_found(err_type('Bind','Describe','Execute') ,type('statement','portal'))`

Reply:
Maybe we should consider clasfiy it as PsqlError:NoStatementFound Error, instead of clasify by phase (Bind, Execute, Describe)...

Reply:
I think clarify by phase is more helpful for developer.